api/server/router/image/image_routes.go

```diff
@@ -205,7 +206,16 @@ func (ir *imageRouter) postImagesPush(ctx context.Context, w http.ResponseWriter
 		ref = r
 	}
 
-	if err := ir.backend.PushImage(ctx, ref, metaHeaders, authConfig, output); err != nil {
+	var platform *ocispec.Platform
+	if platformStr != "" {
```

Comment:
Maybe we need to only look at this if API version is correct?

Reply:
Done, added an error when using before 1.46